enterprise/server/remote_execution/containers/ociruntime/ociruntime.go

```diff
@@ -487,8 +490,8 @@ func (c *ociContainer) createSpec(cmd *repb.Command) (*specs.Spec, error) {
 	return &spec, nil
 }
 
-func (c *ociContainer) invokeRuntimeSimple(ctx context.Context, args ...string) error {
-	res := c.invokeRuntime(ctx, &repb.Command{}, args...)
+func (c *ociContainer) invokeRuntimeSimple(ctx context.Context, stdio *interfaces.Stdio, args ...string) error {
```

Comment:
could we instead use the non-simple func for the callers that want to discard stdio? The (badly named) `Simple` variant was intended to always buffer stderr, and if the command fails, return the buffered stderr in the `error`.